frontend/src/app/modules/project/components/project-settings/project-settings.component.ts

```diff
@@ -73,54 +73,61 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy {
             { label: 'Green', value: '#008000' },
             { label: 'Light Slate Grey', value: '#778899' },
         ]
-  }
+    }
 
-  projectItemIsNotChange(): boolean {
-    return this.projectForm.get('name').value === this.projectStartState.name
-      && this.projectForm.get('description').value === this.projectStartState.description
-      && Number(this.projectForm.get('countOfSaveBuilds').value) === this.projectStartState.countOfSaveBuilds
-      && Number(this.projectForm.get('countOfBuildAttempts').value) === this.projectStartState.countOfBuildAttempts
-      && this.projectForm.get('color').value === this.projectStartState.color;
-  }
+    projectItemIsNotChange(): boolean {
+        return this.projectForm.get('name').value === this.projectStartState.name
+        && this.projectForm.get('description').value === this.projectStartState.description
+        && Number(this.projectForm.get('countOfSaveBuilds').value) === this.projectStartState.countOfSaveBuilds
+        && Number(this.projectForm.get('countOfBuildAttempts').value) === this.projectStartState.countOfBuildAttempts
+        && this.projectForm.get('color').value === this.projectStartState.color;
+    }
 
-  ngOnDestroy() {
-    takeUntil(this.unsubscribe$);
-  }
+    ngOnDestroy() {
+        takeUntil(this.unsubscribe$);
+    }
 
-  onSubmit() {
-    this.isDetailsSaved = false;
-    this.projectService.updateProject(this.project)
-      .subscribe(
-        (resp) => {
-          this.router.navigate([`project/${this.projectId}`]);
-          this.isDetailsSaved = true;
-          this.toastService.success('New details have successfully saved!');
-        },
-        (error) => {
-          this.isDetailsSaved = true;
-          this.toastService.error('Can\'t save new project details', 'Error Message');
-          console.error(error.message);
-        }
-      );
-  }
+    onSubmit() {
+        this.isDetailsSaved = false;
+        this.projectService.updateProject(this.project)
+        .subscribe(
+            (resp) => {
+            this.router.navigate([`project/${this.projectId}`]);
+            this.isDetailsSaved = true;
+            this.toastService.success('New details have successfully saved!');
+            },
+            (error) => {
+            this.isDetailsSaved = true;
```

Comment:
Should be false on error?

Reply:
Nop, it's must be false, this variable is for spinner. I am agree it's not obvious name. I'll change on "HasDetailsSaveResponse"